nemo_skills/inference/generate.py

```diff
@@ -259,10 +245,6 @@ def setup_llm(self):
         return llm
 
     def setup_prompt(self):
-        if self.cfg.prompt_config is None:
```

Comment:
So no more default prompt config ?

Reply:
it's still set by ns eval, so no changes there. The only change is really when you used ns generate with dataset/split parameters but those are removed now. So shouldn't affect any of the use-cases

Reply:
Ah ok that's a good change. Earlier having eval args in generate was confusing